### DISCUSSION TITLE
Database design tool.

### DISCUSSION THREAD

**Discussion by DevCassie** [EddieHubCommunity/support#906] [Tips and Tricks]
Hello friends!

I just stumbled upon a useful link to create a database design, that I wanted to share with you 😄 .
So basically planning what needs to be in the database and what type of input it is etc.

Link: 
https://dbdiagram.io/home

**Comment by eddiejaoude** [comment]
Looks interesting, thanks for sharing 👍

**Reply by hasthamalp** [reply]
thanks for sharing
